doc/connectivity/usb/device_next/usb_device.rst

```diff
@@ -1,57 +1,187 @@
 .. _usb_device_stack_next:
 
-New experimental USB device support
-###################################
+New USB device support
+######################
 
 Overview
 ********
 
-The new USB device support is experimental. It consists of :ref:`udc_api`
-and :ref:`usbd_api`. The new device stack brings support for multiple device
-controllers, support for multiple configurations, and dynamic registration of
-class instances to a configuration at runtime. The stack also provides a specific
-class API that should be used to implement the functions (classes).
-It will replace :ref:`usb_device_stack`.
+USB device support consists of the USB device controller (UDC) drivers
+, :ref:`udc_api`, and USB device stack, :ref:`usbd_api`.
+The :ref:`udc_api` provides a generic and vendor independent interface to USB
+device controllers, and although, there a is clear separation between these
+layers, the purpose of :ref:`udc_api` is to serve new Zephyr's USB device stack
+exclusively.
 
-If you would like to play around with the new device support, or the new USB
-support in general, please try :zephyr:code-sample:`usb-shell` sample. The sample is mainly to help
-test the capabilities of the stack and correct implementation of the USB controller
-drivers.
+The new device stack supports multiple device controllers, meaning that if a
+SoC has multiple controllers, they can be used simultaneously. Full and
+high-speed device controllers are supported. It also provides support for
+registering multiple function or class instances to a configuration at runtime,
+or changing the configuration later. It has built-in support for several USB
+classes and provides an API to implement custom USB functions.
 
-Supported USB classes
-*********************
+The new USB device support is considered experimental and will replace
+:ref:`usb_device_stack`.
 
-Bluetooth HCI USB transport layer
-=================================
+Built-in functions
+==================
+
+The USB device stack has built-in USB functions. Some can be used directly in
+the user application through a special API, such as HID or Audio class devices,
+while others use a general Zephyr RTOS driver API, such as MSC and CDC class
+implementations. The *Identification string* identifies a class or function
+instance (n) and is used as an argument to the :c:func:`usbd_register_class`.
 
-See :ref:`bluetooth-hci-usb-sample` sample for reference.
-To build the sample for the new device support, set the configuration
-``-DCONF_FILE=usbd_next_prj.conf`` either directly or via ``west``.
++-----------------------------------+-------------------------+-------------------------+
+| Class or function                 | User API (if any)       | Identification string   |
++===================================+=========================+=========================+
+| USB Audio 2 class                 | :ref:`uac2_device`      | uac2_(n)                |
```

Comment:
you may want to play with / use the following role:

```suggestion
| USB Audio 2 class                 | :ref:`uac2_device`      | :samp:`uac2_{n}`                |
```

even if you don't end up using it, putting the id strings between double backquotes might make things easier to read.